roadmap/state-sync-protocol.md

```diff
@@ -0,0 +1,198 @@
+# Execution State Synchronization Protocol
+
+## Context
+
+Today, Access nodes act as the gateway to the Flow network, providing a query endpoint for users (REST, gRPC) and performing important admission control duties such as:
+* Rate limiting of requests (script executions, Collection requests, account queries, etc.)
+* Basic validation, preventing bad transactions from entering the network
```

Comment:
Technically, ANs can't _prevent_ bad transactions from entering the network. But they can provide a first level of filtering (imperfect but providing a significant hurdle to spamming the network).